.github/workflows/test-windows.yml

```diff
@@ -82,17 +73,11 @@ jobs:
         with:
           rust: 'true'
 
-      - name: Build profiler
-        run: ./build.bat profiler-zip
-
       - name: Setup Testcontainers Cloud Client
         if: github.event_name != 'pull_request' || github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false
         uses: atomicjar/testcontainers-cloud-setup-action@v1
         with:
           token: ${{ secrets.TC_CLOUD_TOKEN }}
 
       - name: 'Tests: Profiler'
-        uses: ./.github/workflows/test
-        with:
-          name: 'profiler'
-          project: 'test/profiler/Elastic.Apm.Profiler.Managed.Tests/Elastic.Apm.Profiler.Managed.Tests.csproj'
\ No newline at end of file
+        run: ./build.bat test --test-suite Profiler
```

Comment:
ditto :)

Reply:
Thanks @v1v  I don't know why this didn't trigger my code ocd 🙈